.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Python
+
+on:
+  [push, pull_request]
```

Comment:
this should probably be on pushes to `main` only, otherwise it'll run twice for both pull request and push.

Reply:
```suggestion
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```